catroid/src/main/res/values/strings.xml

```diff
@@ -1306,18 +1304,18 @@
 
     <!-- Notifications -->
     <string name="notification_screenshot_ok">Preview image saved</string>
-    <string name="notification_download_project_cancel">Canceled program download</string>
-    <string name="notification_reconvert_download_program_cancel">Canceled downloading/reconverting program</string>
+    <string name="notification_download_project_cancel">Canceled project download</string>
```

Comment:
```suggestion
    <string name="notification_download_project_cancel">Cancel project download</string>
```
This is used as a tappable text in a dialog (overwrite or rename downloaded project) that allows to cancel the download.